example/build.gradle

```diff
@@ -1,5 +1,7 @@
 dependencies {
     implementation project(':starter')
     implementation 'org.springframework.boot:spring-boot-starter-webflux'
+    implementation "org.springdoc:springdoc-openapi-security:1.5.4"
```

Comment:
what requires this dependency?

Reply:
Its requred only by test, i will change it to testImplementation, here is the log:

/Users/markograbuloski/Documents/backendCode/openapi-boot-starter/example/src/test/java/com/vivy/springdoc/scoped/startertest/StarterApiTests.java:4: error: package io.swagger.v3.oas.models does not exist
import io.swagger.v3.oas.models.OpenAPI;
                               ^
/Users/markograbuloski/Documents/backendCode/openapi-boot-starter/example/src/test/java/com/vivy/springdoc/scoped/startertest/StarterApiTests.java:5: error: package io.swagger.v3.oas.models.security does not exist
import io.swagger.v3.oas.models.security.SecurityScheme;
                                        ^
/Users/markograbuloski/Documents/backendCode/openapi-boot-starter/example/src/test/java/com/vivy/springdoc/scoped/startertest/StarterApiTests.java:27: error: cannot find symbol
    private OpenAPI openAPI;
            ^
  symbol:   class OpenAPI
  location: class StarterApiTests
/Users/markograbuloski/Documents/backendCode/openapi-boot-starter/example/src/test/java/com/vivy/springdoc/scoped/startertest/StarterApiTests.java:36: error: package SecurityScheme does not exist
        assertThat(securityScheme.getIn()).isEqualTo(SecurityScheme.In.HEADER);
                                                                   ^